### PR TITLE
docs: added meeting notes

### LINKED ISSUE #31: Admins Meeting 2023-01-24
## Time
- 24/01/2023 14:00 to 14:30 (Madrid time)

## Links
- [Meeting minutes Doc](https://docs.google.com/document/d/15SXIrovYyQjHcaQ-U0UjyxRqF_yc6RtNY03RATsFEpA/edit?usp=sharing)

## Agenda

- #1 
- #21
- #22
- #30 
- #29 
- #28 
- #27
- #20

## Invited

- @onebeyond/admins 
- @onebeyond/maintainers

## Joining the meeting
- [Participants link](https://us04web.zoom.us/j/72448249113?pwd=HNops9PrgYiU3I4rlIQpQOFyc8nF0C.1&from=addon)

### PR DESCRIPTION
**Pull request by UlisesGascon**
Close #31

### CODE REVIEW

File: meetings/2023-01-24.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Announcements
+
+- New Slack dedicated to Open Source!
+- Great support inside the org (Pool results). Should we stream the meeting live?
+
+## Agenda
+
+
+### Repo template as baseline [#1](https://github.com/onebeyond/admin/issues/1)
+- CoC, other health files in progress
```
Comment (inigomarquinez) [inline meetings/2023-01-24.md:25]:
```suggestion
- Added CoC at [the organization level](https://github.com/onebeyond/.github/blob/main/docs/CODE_OF_CONDUCT.md)
- Other [health files](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) in progress 
```

File: meetings/2023-01-24.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- We might want to include some Security basic with sample pipelines like [OpenSSF scorecard](https://securityscorecards.dev/)
+
+### Prototype Pollution in JSON5 via Parse Method [#21](https://github.com/onebeyond/admin/issues/21)
+- No time from Ulises, Iñigo will leader the initiative
```
Comment (inigomarquinez) [inline meetings/2023-01-24.md:29]:
```suggestion
- Ulises doesn't have time, so Iñigo will lead the initiative
```

File: meetings/2023-01-24.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+### Missing NPM Package publication permissions [#28](https://github.com/onebeyond/admin/issues/28)
+- Contact people in charge of those repos
+- Open for collaboration (please comment in the issue if you are interested)
+
+### Unify licenses in Repositories [#27](https://github.com/onebeyond/admin/issues/27)
+- Ulises has been reviewing the licenses in all the repositories and most of them have been patched, but he needs to re-patch to include the license periods for different authors
+
+### Initiatives for 2023 [#20](https://github.com/onebeyond/admin/issues/20)
+
+- We need to involve more people (let's make some noise in Slack)e
```
Comment (inigomarquinez) [inline meetings/2023-01-24.md:55]:
```suggestion
- We need to involve more people (let's make some noise in Slack)
```

**Review by inigomarquinez — CHANGES_REQUESTED** [review]
Some suggestions added

**Comment by UlisesGascon** [comment]
Thanks for all the comments @inigomarquinez :)

**Pull request merged**